nss-day-cohort-22/nutshell-malicious-humans
# Event Page Test

Test to see if you can:
1) add a new event
2) edit an event
3) that the next event is formatted differently (blue background, bold text)
4) if there are multiple events, when you refresh the page they appear in chronological order
5) if you navigate away from the events page and the return to it, then all the events that you entered are still there
6) that you are not able to add an event that has a date prior to today's date
